helm/cluster-aws/values.schema.json

```
@@ -145,6 +145,68 @@
                         }
                     }
                 },
+                "clusterNetwork": {
+                    "type": "object",
+                    "title": "ClusterNetwork",
```

Titles should always be sentence case.

```suggestion
                    "title": "Cluster network",
```